public/index-style.css

```diff
@@ -0,0 +1,182 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+*:focus {
+  outline: none !important;
+}
+
+body {
+  display: grid;
+  width: auto;
+  height: 100vh;
+  grid-template-rows: 5rem auto 5rem;
+}
+
+/*Header //////////////////////////////////////////////////////////*/
+#header {
```

Comment:
Your header could probably be a flexbox instead of a grid since you're only laying things out on a single axis. This would make your page a bit more responsive.